Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM python:3.5
-WORKDIR /code
-ADD . /code
```

Comment:
So you don't build the container from the current code anymore. This is a big change! What advantage do you see?